mesos-client/README.md

```diff
@@ -0,0 +1,128 @@
+# Mesos V1 Client
```

Comment:
I had a discussion with @zen-dog about READMEs in project folders. We should agree on a common pattern. IMHO most of the content here should be in `./docs` instead since it can be subject to change. The readme should link to these docs. What do you htink?

Reply:
Hmm I am not sure. I picture this piece as a standalone library at one point and then having it in readme makes sense. I guess I would probably like to have both but not sure how to do that - so that we don't have to have two copies

Reply:
Yeah, I talked to @zen-dog and @meln1k and lost my arguments. We do have to define what should be in a sub-module README and what should be in the docs but this should be not be a debate for this PR.